reascripts/common/tests/TestStorage.lua

```diff
@@ -43,6 +44,22 @@ function TestStorage:testExtState()
   lu.assertEquals(my_string:get(), 'hello world')
   my_string:erase()
   lu.assertEquals(my_string:get(), 'hello')
+
+  local my_table = settings:table('my_table', {})
+  local my_table_raw = settings:string('my_table', '{}')
+
+  local my_table_value = my_table:get()
+  lu.assertEquals(my_table_value, {})
+  my_table:set({ key = 'value' })
+  lu.assertEquals(my_table:get(), { key = 'value' })
+  lu.assertEquals(my_table_raw:get(), '{"key":"value"}')
+
+  my_table:erase()
+  lu.assertEquals(my_table:get(), {})
+
+  my_table:set({ 1, 2, 3 })
+  lu.assertEquals(my_table:get(), { 1, 2, 3 })
+  lu.assertEquals(my_table_raw:get(), '[1,2,3]')
```

Comment:
just want to confirm my understanding here - the `_raw` version is pointing at the same storage key as a sanity check on the encoding/decoding of the table version?

Reply:
Yes, that's correct - making sure that the generated JSON looks as expected.